src/plugins/raven/sagas/connectionFlow.js

```diff
@@ -88,6 +77,16 @@ export default function* (raven) {
       yield take(connectsChannel);
       attempts = 0;
     } catch (e) {
+      yield swal.fire({
```

Comment:
@efjacobson i would not alert until after 10 attempts

Reply:
done